src/ucp/wireup/stub_ep.h

```diff
@@ -8,34 +8,53 @@
 #ifndef UCP_WIREUP_STUB_EP_H_
 #define UCP_WIREUP_STUB_EP_H_
 
+#include "address.h"
+
 #include <uct/api/uct.h>
 #include <ucp/api/ucp.h>
+#include <ucp/core/ucp_ep.h>
 #include <ucs/datastruct/queue_types.h>
-#include <ucs/type/class.h>
 
 
 /**
  * Stub endpoint, to hold off send requests until wireup process completes.
+ * It is placed instead UCT endpoint before it's fully connected, and for AM
+ * endpoint it also contains an auxiliary endpoint which can send wireup messages.
+ */
+struct ucp_stub_ep {
+    uct_ep_t            super;         /* Derive from uct_ep */
```

Comment:
if you already use doxygen style comments, I would suggest to go all the way and use /**< ... */